.github/workflows/mpi-operator-docker-image-publish.yml

```diff
@@ -0,0 +1,55 @@
+name: build and publish mpi multi-arch docker image
+
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v*"
+  pull_request:
+    branches:
+      - "master"
```

Comment:
Is the intention here that the workflow runs for every PR to build the image, but not push? If so, worth a coment.

As a side note: did you base this configuration on some other repo?

Reply:
yes, that's the intention and I'll add the comment for the same.
I started out with example from this github action.
[docker meta action](https://github.com/docker/metadata-action)